iceoryx_examples/icedelivery/iox_subscriber_untyped.cpp

```diff
@@ -39,23 +40,38 @@ int main()
     auto signalTermGuard = iox::posix::registerSignalHandler(iox::posix::Signal::TERM, sigHandler);
 
     // initialize runtime
+    //! [runtime initialization]
+    constexpr char APP_NAME[] = "iox-cpp-subscriber-untyped";
     iox::runtime::PoshRuntime::initRuntime(APP_NAME);
+    //! [runtime initialization]
 
     // initialized subscriber
+    //! [create subscriber untyped]
     iox::popo::UntypedSubscriber subscriber({"Radar", "FrontLeft", "Object"});
+    //! [create subscriber untyped]
 
     // run until interrupted by Ctrl-C
+    //! [loop]
     while (!killswitch)
     {
-        subscriber.take()
+        subscriber
+            .take()
+            //! [chunk happy path]
             .and_then([&](const void* userPayload) {
+                //! [and_then]
```

Comment:
is the `and_then` snippet really necessary?

Reply:
Well, I basically used it to elide everything inside the `and_then`path when the while loop is described in the readme (l. 163...). Otherwise the comments regarding the release of the release of the chunk would be visible. I first decided against wrapping those comments inside the release tags since it is already described in the readme, but now I think the `and_then` snippet reduces the readability of the example...

Reply:
I think the comment would also be valuable in the README, maybe a little bit rephrased. The README itself only mention the release and not that a later access will be undefined behavior.